### PR TITLE
Add error boundaries for analysis tabs

### PR DESCRIPTION
**Pull request by chowington**
Fixes https://github.com/VEuPathDB/web-eda/issues/365

### CODE REVIEW

**Comment by chowington** [comment]
I added error boundaries to just the routes in the analysis panel that rendered tabs. Let me know if you think I should add boundaries to some of the other routes, too.

**Comment by chowington** [comment]
You can use the first commit above to test with an error; it has an artificial error in the subsetting component.

File: src/lib/workspace/AnalysisPanel.tsx [hunk 1]
```diff
@@ -30,12 +30,32 @@ import { EntityDiagram } from '../core';
...
+  children,
+}: {
+  children: React.ReactNode;
+}) => (
+  <ErrorBoundary
+    children={children}
+    renderError={() => (
+      <Alert severity="error" style={{ marginTop: 10 }}>
+        We're sorry, something went wrong with this tab. Please change to
+        another tab or refresh to try again.
```
Comment (jtlong3rd) [inline src/lib/workspace/AnalysisPanel.tsx:54]: I'd offer a link to let the user reload the tab, similar to [what our (in?)famous "oops" modal offers](https://github.com/VEuPathDB/EbrcWebsiteCommon/blob/bc5dd3077310a46457e95d85f39f341114d644a6/Client/src/component-wrappers/Error.tsx#L39).

```suggestion
        We're sorry, something went wrong with this tab. Please change to
        another tab or {<a href="" title="Reload the current tab.">reload this tab</a>} to try again.
```

**Review by jtlong3rd — APPROVED** [review]
I have one small inline suggestion, but this should otherwise be good to merge!

**Pull request merged**